### PR TITLE
New db attributes

### PR DESCRIPTION
**Pull request by aleks279**
Add several models and model validation

This will require you to run the next `rake` commands:

`rake db:drop:all`
`rake db:create:all`
`rake db:migrate`

Those will recreate the data base. Run this other, just to make sure everything is ok ->

`rspec`

### CODE REVIEW

File: db/migrate/20160401165621_create_genre.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class CreateGenre < ActiveRecord::Migration
+  def change
+    create_table :genres do |t|
+      t.string :name
```
Comment (jramoscr) [inline db/migrate/20160401165621_create_genre.rb]: Add `null: false` to this line.

File: db/migrate/20160401165639_create_event_type.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+class CreateEventType < ActiveRecord::Migration
+  def change
+    create_table :event_types do |t|
+      t.string :name
```
Comment (jramoscr) [inline db/migrate/20160401165639_create_event_type.rb]: Same comment about `null: false`.

File: db/migrate/20160401171705_create_instrument.rb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+class CreateInstrument < ActiveRecord::Migration
+  def change
+    create_table :instruments do |t|
+      t.string :name
```
Comment (jramoscr) [inline db/migrate/20160401171705_create_instrument.rb]: `null: false`

File: db/migrate/20160401171710_create_instrument_type.rb [hunk 4]
```diff
@@ -0,0 +1,9 @@
+class CreateInstrumentType < ActiveRecord::Migration
+  def change
+    create_table :instrument_types do |t|
+      t.string :name
```
Comment (jramoscr) [inline db/migrate/20160401171710_create_instrument_type.rb]: `null: false`

**Comment by cardoce** [comment]
Great idea adding genres to musicians and bands, the event types confuse me a bit tho, is it suppose to be use to divide by weddings, concerts, live music at bars, and things like that?

**Comment by aleks279** [comment]
@cardoce correct. The event types manage 2 things, the type of an event (duh!) and the event type (let's say purpose) a musician or band has: e.i. a band plays only for weddings, or a musician plays for birthday parties and bars, etc.

Somewhat similar to the genres

**Comment by cardoce** [comment]
@aleks279 ohh I see! Thanks!

**Comment by 4RN4LDO** [comment]
@aleks279 looks good to me :metal:

**Pull request merged**